### PR TITLE
Run the Behat acceptance tests against the skeune theme

### PR DESCRIPTION
**Pull request by MKodde**
The aim of this PR is to run the acceptance tests against the Skeune theme in al its glory. Including translations, and theme specific settings and optimizations.

This allows us to create new tests carelessly in the future. As the new theme should become the new default in the near future.

### CODE REVIEW

File: .github/workflows/test-integration.yml [hunk 1]
```diff
@@ -83,7 +83,9 @@ jobs:
                       echo -e "\nInstalling database fixtures...\n" && \
                       ./app/console doctrine:schema:drop --force --env=ci && \
                       ./app/console doctrine:schema:create --env=ci && \
-                      echo -e "\nBehat tests\n" && \
+                      echo -e "\nPreparing frontend assets\n" && \
+                      EB_THEME=skeune ./theme/scripts/prepare-test.js > /dev/null && \
```
Comment (MKodde) [inline .github/workflows/test-integration.yml:87]: In order to run the tests against an actual Skeune EngineBlock, an additional run of the build was required. The perpare test also cache-clears the application. So a fresh env is used for the Behat tests.

An additional, but required time penalty for the tests to run..

File: app/config/config_test.yml [hunk 2]
```diff
@@ -11,8 +11,11 @@ services:
 framework:
     test: ~
     translator:
+        fallbacks: ["%locale%"]
         paths:
             - '%kernel.root_dir%/../languages'
+            - '%kernel.root_dir%/../theme/base/translations'
+            - '%kernel.root_dir%/../theme/%theme.name%/translations'
```
Comment (MKodde) [inline app/config/config_test.yml:18]: In order to get the correct translations in test, also load the theme translation overrides here.

File: app/config/functional_testing.yml.dist [hunk 3]
```diff
@@ -5,3 +5,5 @@ parameters:
     # Where must we store the writable state of the Mock IdP and Mock SP?
     idp_fixture_file:   "%kernel.root_dir%/../tmp/fixtures/db/idp.states.php.serialized"
     sp_fixture_file:    "%kernel.root_dir%/../tmp/fixtures/db/sp.states.php.serialized"
+
+    theme.name: "skeune"
```
Comment (MKodde) [inline app/config/functional_testing.yml.dist:9]: Ensure the test/ci environment is set with the Skeune theme by default. The test and CI envs  include this additional parameters file, overwriting parameters.dist settings for the test environments.

**Review by MKodde — COMMENTED** [review]
See some additional information as to why I had to undertake some of the more piquant changes.

Enjoy!

**Review by Badlapje — APPROVED** [review]
Looking good, great work!  Loved the short notes you gave at the start 💯

**Comment by MKodde** [comment]
Merging

**Pull request merged**